charts/gha-runner-scale-set/values.yaml

```diff
@@ -170,3 +170,7 @@ template:
 # controllerServiceAccount:
 #   namespace: arc-system
 #   name: test-arc-gha-runner-scale-set-controller
+
+kubernetesMode:
```

Comment:
I think that the better place for this object would be under `containerMode` object, similar to the [`kubernetesModeWorkVolumeClaim`](https://github.com/actions/actions-runner-controller/blob/f1d7c52253b89f0beae60141f8465d9495cdc2cf/charts/gha-runner-scale-set/values.yaml#L80)
It can be named something like `kubernetesModeServiceAccount`
What do you think @Link- ?

Reply:
@nikola-jokic  I agree. Opinion reflected.